org.eclipse.draw2d/src/org/eclipse/draw2d/FanRouter.java

```diff
@@ -67,11 +68,11 @@ protected void handleCollision(PointList points, int index) {
 		Point bendPoint;
 
 		if (index % 2 == 0) {
-			bendPoint = new Point(midPoint.x + (int) ((index / 2.0) * -1.0 * ySeparation),
-					midPoint.y + (int) ((index / 2.0) * xSeparation));
+			bendPoint = new PrecisionPoint(midPoint.x + (index / 2) * (-1 * ySeparation),
```

Comment:
I think we should move (index/2) to a stack variable. Would make for me the code better readable, and the I wouldn't have made the wrong clean-up if it would have been that way.

Reply:
Instead of having a separate variable, I can also just have it be part of the calculation for the `xSeparation` and `ySeparation`.

Reply:
Good idea.